lib/database_connection.rb

```diff
@@ -0,0 +1,37 @@
+require 'pg'
+require 'uri'
+
+class DatabaseConnection
+
+  def self.setup
```

Comment:
This method is long: it is the method where the DatabaseConnection class decides which database to connect to and unfortunately there are three options: test, development, or production. The production connection uses a URI from heroku which needs to be parsed. This could/should be split out into a separate method.